src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -134,17 +137,19 @@ public void testExceptionalCases() throws Exception {
 
         resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.SanitizedTeam.instr",
                                                            "Session with sanitized data");
-
+        clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-1", "ajax_auto");
         resultsPage.verifyHtmlMainContent("/instructorFeedbackResultsPageWithSanitizedData.html");
     }
 
     private void testModerateResponsesButton() {
 
         resultsPage = loginToInstructorFeedbackResultsPage("CFResultsUiT.instr", "Open Session");
         resultsPage.displayByQuestion();
-
+        clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-2", "ajax_auto");
+        clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-4", "ajax_auto");
         ______TS("Typical case: test moderate responses button for individual response (including no response)");
 
+        ThreadHelper.waitFor(1000);
```

Comment:
I think this can be safely removed for now as the panel wait function is updated. Thx for the suggestion!